README.md

```diff
@@ -56,19 +56,19 @@ Live Server
 ### Setup
 
 Clone this repository to your desired folder: 
-git@github.com:Gmanlove/Hello-Microverse.git
+https://github.com/tomdan-ai/Hello-Microverse.git
 
 
 ## 👥 Authors <a name="authors"></a>
 
 👤 **Author1**
 
-- GitHub: [Gmanlove](https://github.com/Gmanlove)
+- GitHub: [tomdan-ai](https://github.com/tomdan-ai)
 
 
```

Comment:
- [ ] Kindly include 1-3 features that can be implemented in the **_Future Features_** section. Features like building a webpage, navigation or whatever you desire. 👍